tests/test_domain_cpp.py

```diff
@@ -1015,7 +1015,7 @@ def test_build_domain_cpp_semicolon(app, status, warning):
 
 @pytest.mark.sphinx(testroot='domain-cpp',
                     confoverrides={'nitpicky': True, 'strip_signature_backslash': True})
-def test_build_domain_cpp_backslash_ok(app, status, warning):
+def test_build_domain_cpp_backslash_ok2(app, status, warning):
```

Comment:
@jakobandersen This is also.

Reply:
This one should probably be called ``test_build_domain_cpp_backslash_ok_true`` and the previous one ``test_build_domain_cpp_backslash_ok_false``, i.e., encoding the value of ``strip_signature_backslash`` in the name.